GRAMMAR.md

```diff
@@ -61,4 +61,9 @@ call ::= args | ':' Name args
 suffix ::= call | index
 var ::= prefix {suffix} index | Name
 functioncall ::= prefix {suffix} call
+```
+
+Roblox specific grammar
```

Comment:
We don't really update this file--I think it was mostly for reference. Plus, this isn't correct to begin with since you don't use this anywhere outside of defining it.